testapp/src/main/res/layout/settings_menu_base_activity.xml

```diff
@@ -0,0 +1,74 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:padding="@dimen/medium_16"
+  android:orientation="vertical">
+
+  <androidx.appcompat.widget.AppCompatTextView
+    android:id="@+id/textInfo"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content" />
+
+  <RelativeLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/small_8">
```

Comment:
This layout could be a top toolbar with a save action on Android. Adapt iOS features to what you know on Android, it does not have to be 100% identical